ncsa_teststand/lsst-teststand-ts1/shutdown_love

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+source /deploy-lsstts/.love_secrets
```

Comment:
You may want to stick to a common naming convention. Here the `.love_secrets` file doesn't have an extension but in the amos01 script it does.